frontend/src/components/Pages/MissionPage.tsx

```diff
@@ -34,6 +34,20 @@ export function MissionPage() {
         }
     }, [])
 
+    useEffect(() => {
+        const id = setInterval(() => {
+            if (missionId) {
+                apiCaller.getMissionById(missionId).then((mission) => {
+                    setSelectedMission(mission)
+                    apiCaller.getVideoStreamsByRobotId(mission.robot.id).then((streams) => {
+                        setVideoStreams(streams)
+                    })
+                })
+            }
+        }, 1000)
```

Comment:
I see a magic number, can we first assign this to a const with an explaining name and then use the const here instead?